backend/lib/calculate/adequacy.py

```diff
@@ -15,7 +15,7 @@
 ONE_MILE_IN_METERS = 1609.344
 ONE_METER_IN_MILES = 1.0 / ONE_MILE_IN_METERS
 # Adequacy results will be inaccurate with smaller RELEVANCY_RADIUS_IN_METERS values.
-RELEVANCY_RADIUS_IN_METERS = 15.0 * ONE_MILE_IN_METERS
+RELEVANCY_RADIUS_IN_METERS = 80.0 * ONE_MILE_IN_METERS
```

Comment:
?

Reply:
This is the fix I alluded to -- when service areas were ZIP codes, we only needed to consider providers close to the ZIP code. But now that they're much larger, points in separate counties weren't considering points 45 miles away (but in another county).